frontend/src/metabase-types/api/mocks/actions.ts

```diff
@@ -99,3 +101,24 @@ export const createMockPublicAction = (
   parameters: [],
   ...opts,
 });
+
+export const createMockActionFormSettings = (
+  opts?: Partial<ActionFormSettings>,
+): ActionFormSettings => ({ ...opts });
```

Comment:
nit: would it be simpler to just return the `opts` at this point as you don't add any default values here?

```suggestion
): ActionFormSettings => opts;
```

Also, from a brief text search,  this seems to be the only place that spread the object into a new one without adding or modifying any other property.